rest/config.go

```diff
@@ -199,7 +198,6 @@ type DbConfig struct {
 	DeprecatedRevCacheSize    *uint32                        `json:"rev_cache_size,omitempty"`               // Maximum number of revisions to store in the revision cache (deprecated, CBG-356)
 	StartOffline              bool                           `json:"offline,omitempty"`                      // start the DB in the offline state, defaults to false
 	Unsupported               db.UnsupportedOptions          `json:"unsupported,omitempty"`                  // Config for unsupported features
-	Deprecated                DeprecatedOptions              `json:"deprecated,omitempty"`                   // Config for Deprecated features
```

Comment:
It might be useful to leave a placeholder for future deprecated options.  Can we just remove Shadow from DeprecatedOptions and leave it as an empty struct?